docs/conf.py

```diff
@@ -30,6 +30,7 @@
 
 # default settings
 templates_path = ["_templates"]
+html_static_path = ["_static"]
```

Comment:
I'm not sure if this actually does anything

Reply:
I think it needs to be set. We didn’t have files, so didn’t need it